### DISCUSSION TITLE
In the nuxt3 development environment, window.crypto.subtle cannot be used on the client

### DISCUSSION THREAD

**Discussion by atonemi** [nuxt/nuxt#24966] [Questions] [answered]
How come window.crypto.subtle is undefined in the Nuxt 3 development environment, but it works in the production environment? Can someone explain the reason behind this?
![image](https://github.com/nuxt/nuxt/assets/75244754/2488f7fa-b72d-45f4-8a9d-9c2730bbc971)

**Comment by TheAlexLichter** [comment, marked as answer]
From [MDN](https://developer.mozilla.org/en-US/docs/Web/API/Crypto/subtle):

> Secure context: This feature is available only in [secure contexts](https://developer.mozilla.org/en-US/docs/Web/Security/Secure_Contexts) (HTTPS), in some or all [supporting browsers](https://developer.mozilla.org/en-US/docs/Web/API/Crypto/subtle#browser_compatibility).

**Reply by atonemi** [reply]
I made a low-level and silly mistake: I was trying to access the Nuxt project using http://Intranet ip:port, and after a while, I realized my error. Thanks for the response
